samples/zephyr/subsys/mgmt/mcumgr/smp_svr/sysbuild/mcuboot/boards/nrf54lm20dk_nrf54lm20a_cpuapp_larger_mcuboot.conf

```diff
@@ -0,0 +1,7 @@
+# increasead for encryption ed25519
+CONFIG_PM_PARTITION_SIZE_MCUBOOT=0x14000
+
+# from mcuboot/boot/zephyr/boards
+CONFIG_SPI_NOR=n
+CONFIG_FPROTECT=n
```

Comment:
I do not think this will be necessary with LM20 - it has a much larger region protection area